template/src/features/counter/Counter.js

```diff
@@ -24,7 +24,10 @@ export function Counter() {
         >
           +
         </button>
-        <span className={styles.value}>{count}</span>
+        <span className={styles.value}>
+          <span className="sr-only">Count is {count}</span>
+          {count}
```

Comment:
The below would be ideal because right now screen readers will read the count twice.
```suggestion
          <span className="sr-only">Count is</span>
          {count}
```

But this would also work if you can't easily test the above
```suggestion
          <span className="sr-only">Count is {count}</span>
          <span aria-hidden>{count}</span>
```

Reply:
The first method you suggested was tricky to get the value, as it was in a separate element.

Reply:
I've force pushed with changes that use `getByTitle`  rather than adding css - how does that look to you @BenLorantfy ?

Reply:
I did some research on this and I don't think `title` is recommended
> Short answer: Don’t use them, except in special circumstances.
https://a11yproject.com/posts/title-attributes/

What do you think of this: 
```
<output aria-label="Counter value" aria-live="polite">
  5
</output>
```

and then:
`screen.getByRole('status', { name: /counter value/i })`

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/output
See: [Playground link](
https://testing-playground.com/?markup=DwewrgLgDpAECGAnAlvAtAG3gIwKYYF4AiAYXADsJdFYA3eDMXIgPgCgBWN4AenGkgsgA&query=M4YwTgphB2B0DmEAuAhAngJQPYBsIAoByYJAQyQFdhCAaAAgG87pSBbCALjoHoQsLoSCGDoA3UjgoRuASzoBfAJRA)

Reply:
LGTM. Done